src/components/DetailsPage.js

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable camelcase */
+// the API data dosent use camel case!
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+
+function Region({
+  name, todayConfirmed, todayDeaths, todayOpenCases, todayRecovered,
+}) {
+  return (
+    <li className="detailsRow">
+      <div>
+        <h3>{name}</h3>
+      </div>
+      <div>
+        <h4>
+          Confirmed:
+          {' '}
+          {todayConfirmed}
+        </h4>
+        <h4>
+          Deaths:
+          {' '}
+          {todayDeaths}
+        </h4>
+        <h4>
+          Open Cases:
+          {' '}
+          {todayOpenCases}
+        </h4>
+        <h4>
+          Recovered:
+          {' '}
+          {todayRecovered}
+        </h4>
+      </div>
+    </li>
+  );
+}
+
+Region.propTypes = {
+  name: PropTypes.string.isRequired,
+  todayConfirmed: PropTypes.number.isRequired,
+  todayDeaths: PropTypes.number.isRequired,
+  todayOpenCases: PropTypes.number.isRequired,
+  todayRecovered: PropTypes.number.isRequired,
+};
```

Comment:
- [ ] Optional suggestions

-  PropType exports a range of validators that can be used to make sure the data you receive is valid. I would recommend including/adding it in the below lines after a return to make our code look clean and clear.